src/App.vue

```diff
@@ -1,14 +1,16 @@
 <template>
   <v-app id="app">
-    <navigation-drawer v-if="showNav" />
+    <the-top-navbar :isSmallDevice="isSmallDevice" v-if="showNav" />
```

Comment:
This will start being fixed automatically by the linter after #526 is merged, but v-if goes before props according to the vue style guide
```suggestion
    <the-top-navbar v-if="showNav" :isSmallDevice="isSmallDevice" />
```